frontend/src/components/occupationcard.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+export default function OccupationCard({ occupation }) {
+  if (!occupation) {
+    return <div>Loading...</div>;
+  }
+
+  return (
+    <div className="occupation-card">
+      <h2>{occupation.name} (Code: {occupation.stdCode})</h2>
+      <p><strong>Status:</strong> {occupation.statusName} (Level {occupation.level}, Version {occupation.versionNo})</p>
+      <p><strong>Overview:</strong> {occupation.overview}</p>
+      <div>
+        <strong>Summary:</strong>
+        <div dangerouslySetInnerHTML={{ __html: occupation.summary }} />
```

Comment:
what did we say about pushing code we don't understand? why don't we just interpolate the summary into the code like we're doing with the other values here?